src/main/java/net/pms/PMS.java

```diff
@@ -880,60 +875,52 @@ public static void main(String[] args) {
 		boolean displayProfileChooser = false;
 		boolean denyHeadless = false;
 		File profilePath = null;
-
+		checkCompatibilityWithJavaVersion();
```

Comment:
I think this line and the method it calls are the only changes in this file, can you revert the other changes?